aviator-cli/rebasing-and-syncing-with-github.md

```diff
@@ -0,0 +1,64 @@
+# Rebasing and Syncing with GitHub
+
+When you work on a stack of branches, the state of the stack can be out of sync
+with the remote or a parent branch. For example, if the `main` branch on the
+remote gets updates by merging other pull-requests, or a new commit is added or
+a commit is amended in one of the parent branches, or some of the parents are
+merged, you want to rebase your branches on top of the new commits.
+
+`av stack sync` is an action that does that for you. Aviator CLI fetches the
```

Comment:
We should mention both stack sync and stack restack here as two options to do that.